Wikipedia/UI-V5/WMFArticleContainerViewController.h

```diff
@@ -1,18 +1,22 @@
-
 @import UIKit;
+#import "WMFArticleContentController.h"
+#import "WMFArticleListItemController.h"
 
 @class WMFArticleViewController;
 @class MWKDataStore;
 @class MWKSavedPageList;
 @class MWKArticle;
 
-@interface WMFArticleContainerViewController : UIViewController
+NS_ASSUME_NONNULL_BEGIN
 
-+ (instancetype)articleContainerViewControllerWithDataStore:(MWKDataStore*)dataStore savedPages:(MWKSavedPageList*)savedPages;
+@interface WMFArticleContainerViewController : UIViewController
+    <WMFArticleContentController, WMFArticleListItemController>
 
-@property (nonatomic, strong) MWKArticle* article;
```

Comment:
It was moved into a protocol: `WMFArticleContentController` which the container, article & webVC's all conform to